unikernel_create.ml

```diff
@@ -1,4 +1,9 @@
-let unikernel_create_layout =
+let unikernel_create_layout ~(user_policy : Vmm_core.Policy.t) unikernels
+    (blocks : (Vmm_core.Name.t * int * bool) list) =
+  let _, _, _, total_memory_used, count_cpuid_usage, _ =
+    Utils.user_policy_usage user_policy unikernels blocks
+  in
```

Comment:
as mentioned above, I'd prefer a record.

Reply:
and now that only two values are used, I wonder whether it makes sense to split "Utils.user_policy_usage" into smaller functions?